.changeset/big-cycles-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lynx-js/web-elements": patch
+---
+
+feat: add support for `scrollTo` method in x-swiper
```

Comment:
---
"@lynx-js/web-elements": patch
---

feat: add support for `scrollTo` method in x-swiper, remove `scrollToNext` && `scrollToPrevious` method